src/main/resources/application.yml

```diff
@@ -0,0 +1,13 @@
+spring:
```

Comment:
Seems good, just add `@EnableEurekaClient` to the class that hold the `main` method so that it will know that this is going to be a client for the Eureka Server